src/mlpack/methods/elm/elm.hpp

```diff
@@ -0,0 +1,51 @@
+/**
+ * @file elm.hpp
+ * @author Siddharth Agrawal
+ *
+ * Basic Extreme Learning Machine
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_ELM_HPP
+#define MLPACK_METHODS_ELM_HPP
+
+#include <mlpack/core.hpp>
+
+namespace mlpack {
+namespace elm {
+
+class ELM
+{
+private:
+	uint16_t N_prime; //Number of Hidden Neurons
+	uint16_t Dim; //Data Dimension
+	uint16_t NI; //Number of data points
+	mat Weight;
+	vec bias;
+	mat beta;
+	uint16_t Activation;
+
+	double Train_time;
+	double Test_time;
+
+	double Train_Accuracy;
+	double Test_Accuracy;
+
+public: 
+	void Set_Dim(uint16_t Nh, uint16_t D, uint16_t N);
```

Comment:
Besides, please write test cases for your class.You can use [softmax as example](https://github.com/mlpack/mlpack/blob/master/src/mlpack/tests/softmax_regression_test.cpp) too.